vanilla-installer/main.py

```diff
@@ -4,20 +4,22 @@
 # IMPORTS
 
 # Standard library
+import io
+import json
 import os
 import sys
 import logging
 import logging.handlers  # pylance moment
-import re
 import subprocess
-import tempfile
 import pathlib
 import base64
+import zipfile
 
 # External
 import requests
 import minecraft_launcher_lib as mll
 import click
+import tomllib # you should consider add a toml parser to project, or limit installer to py 3.11
```

Comment:
We'll probably just have to use a third-party library, since limiting the installer to 3.11 isn't feasible right now according to [#15 (comment)](https://github.com/Fabulously-Optimized/vanilla-installer/issues/15#issuecomment-1276310170). I'll likely change this after this PR is merged though.

Reply:
Well, you can just change the PR